javascript/javascript2/week1/Javascript2Project/functionality.js

```diff
@@ -1,32 +1,21 @@
-// Creating a card deck from cards and suits
-function createCardDeck() {
-  cardsArray.length = 0;
-  cards.forEach((deckCard) => {
-    suits.forEach((suit, index) => {
-      cardsArray.push({
-        cardName: `${deckCard} of ${suit}`,
-        cardUrl: `./cards/${deckCard}${suitsFirstLetter[index]}.webp`,
-      });
-    });
-  });
-}
-
 // Generate random cards from the deck, the amount of cards is determined by user in gameSize
 function generateRandomCard() {
   if (randomCardsArray.length) return;
 
-  let shuffledCards = [...cardsArray].sort(() => Math.random() - 0.5); // Shuffle the deck
+  const shuffledCards = [...cardsArray].sort(() => Math.random() - 0.5);
```

Comment:
It should be fine to remove this shuffling since you do it on `randomCardsArray` afterwards.